qCC/ccCommandLineCommands.cpp

```diff
@@ -5875,7 +5905,19 @@ bool CommandSFOperation::process(ccCommandLineInterface& cmd)
 		sf2.isConstantValue = true;
 		sf2.constantValue = value;
 	}
-	
+
+	//in place modifier, to keep old commands intact we should keep it in place by default. However it makes the command line inconsistence, because the SF_ARITHMETIC works the other way.
```

Comment:
```suggestion
	//in place modifier, to keep old commands intact we should keep it in place by default. However it makes the command line inconsistent, because the SF_ARITHMETIC works the other way.
```